src/core/provider/ModuleProvider.js

```diff
@@ -0,0 +1,22 @@
+// @flow
+import type { Node } from 'react';
+
+import { ConnectedRouter } from 'connected-react-router/immutable';
+import { Provider } from 'react-redux';
+
+import { moduleContext, moduleStore } from '../redux';
+import { routerHistory } from '../router';
+
+type Props = {
+  children :Node;
```

Comment:
Recent versions of flow will complain about this. Go with `:any` for now and we'll figure it out later.